src/VectorizedSolver.cpp

```diff
@@ -322,76 +328,52 @@ void VectorizedSolver::normalizeFluxes() {
 void VectorizedSolver::computeFSRSources() {
 
   int tid;
-  FP_PRECISION scatter_source;
-  FP_PRECISION fission_source;
-  FP_PRECISION* nu_sigma_f;
-  FP_PRECISION* sigma_s;
-  FP_PRECISION* sigma_t;
-  FP_PRECISION* chi;
   Material* material;
+  FP_PRECISION* sigma_t;
+  FP_PRECISION* sigma_s;
+  FP_PRECISION* fiss_mat;
+  FP_PRECISION scatter_source, fission_source;
 
-  int size = _num_FSRs * _num_groups * sizeof(FP_PRECISION);
+  int size = _num_threads * _num_groups * sizeof(FP_PRECISION);
```

Comment:
Can someone explain to me why we are making the memory size dependent on the number of threads? Is each thread getting a copy of its own data? And do we really need to do that?